knowledge_repo/app/static/css/custom.css

```diff
@@ -256,29 +241,6 @@ a.label:focus, a.label:hover {
   color: #9CA299;
 }
 
-.feed-author {
-  padding-bottom: 5px;
```

Comment:
Got it. Given that this is pending based on future work happening, I think it probably makes sense to hold of on changing `feed-post` to `index-post`.

Fair point, i didn't realize you had re-arranged the css from one file to another.